catalogs/catalog-hadoop/src/main/java/com/datastrato/gravitino/catalog/hadoop/HadoopCatalogOperations.java

```diff
@@ -53,6 +53,9 @@
 
 public class HadoopCatalogOperations implements CatalogOperations, SupportsSchemas, FilesetCatalog {
 
+  private static final String SCHEMA_DOES_NOT_EXIST = "Schema %s does not exist";
```

Comment:
Can you please rename all the string to "XXX_MSG", add "_MSG" suffix to all these string variable names.

Reply:
@jerryshao Thank you for reviewing. I have updated it.